src/metabase/types.clj

```diff
@@ -0,0 +1,94 @@
+(ns metabase.types)
```

Comment:
Yeah, Clojure requires you to namespace keywords when you create a global hierarchy. So we have to do something like stick `:type/` in front of them.